Library/Formula/ceph.rb

```diff
@@ -0,0 +1,60 @@
+class Ceph < Formula
+  desc "Ceph is a scaleable distributed file system. Client tools/libraries only."
+  homepage "http://ceph.com/"
+  head "https://github.com/ceph/ceph.git"
+
+  depends_on "libtool" => :build
+  depends_on "automake" => :build
+  depends_on "autoconf" => :build
+  depends_on "pkg-config" => :build
+  depends_on "openssl"
+  depends_on "snappy"
+  depends_on "cryptopp"
+  depends_on "boost"
+  depends_on "leveldb"
+
+  # Unfortunately it installs a Python script unconditionally, we depend on the
+  # Homebrew python so that when that script gets installed it gets installed
+  # into the Homebrew python site-packages rather than failing to install it in
+  # the system python site-packages.
```

Comment:
@tdsmith That line worked :-). Updated the patchset.

@mikemcquaid Let me know if that is what the decision is. Ultimately I think it would be nice to have it in tree, but that may come down the line.